bin/jruby.sh

```diff
@@ -505,7 +505,21 @@ fi
 
 # ----- Detect Java version and determine available features ------------------
 # shellcheck source=/dev/null
-java_version=$(. "$JAVA_HOME/release" && echo "${JAVA_VERSION-}")
+if [ -f "$JAVA_HOME/release" ]; then
+    java_version=$(. "$JAVA_HOME/release" && echo "${JAVA_VERSION-}")
+    if echo "${java_version}" | grep -q "1.8.\\.*"; then
+        java_version=8
+    fi
+else
+    java_version=8
+fi
+# shellcheck source=/dev/null
+if [ -f "$JRUBY_HOME/bin/.java-version" ]; then
+    minimum_java_version=$(. "$JRUBY_HOME/bin/.java-version" && echo "${JRUBY_MINIMUM_JAVA_VERSION-}")
```

Comment:
I think we can just source this file directly since it's from a trusted source
```sh
if [ -f "$JRUBY_HOME/bin/.java-version" ] && . "$JRUBY_HOME/bin/.java-version" && [ "${JRUBY_MINIMUM_JAVA_VERSION-}" ]; then
    minimum_java_version=$JRUBY_MINIMUM_JAVA_VERSION
else
    # Only 9.4.12.0 and earlier will have shipped without a .java-version file, so fall back on minimum of 8
    minimum_java_version=8
fi
```

Reply:
We could also stick this filename in a variable somewhere to avoid repeating ourselves but not a big deal

Reply:
Fixed in subsequent commit.